src/components/profileDetails.vue

```diff
@@ -1,99 +1,99 @@
 <template>
-    <div>
-      <div class="block place-content-center">
```

Comment:
sound like unecessary changes. please use prettier to format code as it was used in the repository.

Reply:
This file is unnecessary. Nothing was changed.